completers/nix-shell_completer/cmd/root.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"github.com/rsteube/carapace"
+	"github.com/rsteube/carapace-bin/pkg/actions/tools/nix"
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "nix-shell",
+	Short: "start an interactive shell based on a Nix expression",
+	Long:  "https://nixos.org/manual/nix/stable/command-ref/nix-shell.html",
+	Run:   func(cmd *cobra.Command, args []string) {},
+}
+
+func Execute() error {
+	return rootCmd.Execute()
+}
+func init() {
+	carapace.Gen(rootCmd).Standalone()
+
+	rootCmd.Flags().StringSlice("arg", []string{}, "Pass [name] [expr] to Nix functions.")
+	rootCmd.Flags().StringSlice("argstr", []string{}, "Pass [name] [string] to Nix functions")
+	rootCmd.Flags().Bool("help", false, "Show usage information")
+	rootCmd.Flags().StringP("include", "I", "", "Include paths")
+	rootCmd.Flags().String("command", "", "Run the command in an interactive shell")
+	rootCmd.Flags().String("run", "", "Run the command in a non-interactive shell")
+	rootCmd.Flags().String("exclude", "", "Do not build any dependencies whose store path matches the regexp")
+	rootCmd.Flags().Bool("pure", false, "Clear most of the environment for the shell")
+	rootCmd.Flags().String("keep", "", "Environment variable to keep in --pure")
+	rootCmd.Flags().StringSliceP("packages", "p", []string{}, "Packages to add from nixpkgs")
+	rootCmd.Flags().StringP("expr", "E", "", "Nix expression to build")
```

Comment:
These flags are mutually exclusive, is there some way to indicate this in carapace?

Reply:
```go
rootCmd.MarkFlagsMutuallyExclusive("packages", "expr")
```